examples/c-nanopb/proto/example/v1/sensor.proto

```
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+package sensor.v1;
```

Buf (1.54.0)

Files with package "sensor.v1" must be within a directory "sensor/v1" relative to root but were in directory "examples/c-nanopb/proto/example/v1".

(PACKAGE_DIRECTORY_MATCH)